GUI/src/main/kotlin/ui/components/selectVideoScreen/FileSelectorButton.kt

```diff
@@ -29,6 +29,7 @@ import ui.components.general.openFileChooserAndGetPath
  */
 @Composable
 fun RowScope.FileSelectorButton(
+    modifier: Modifier?,
```

Comment:
This argument is never explicitly used in the FileSelectorButton body? Does it actually make a difference? Also, usually modifier arguments are optional, so I would suggest writing `modifier: Modifier? = null`. Then, we could remove the modifier argument for some of the usages :)